test/types/range/lydia/badRange-string-multi.good

```diff
@@ -0,0 +1,2 @@
+badRange-string-multi.chpl:1: In function 'main':
+badRange-string-multi.chpl:2: error: Range bounds must be integers of compatible types
```

Comment:
@lydia-duncan: I can take on the task of rewording this error message (and others that share wordings suggesting ranges only support ints) if it bogs down this PR.  I just ran into it in some other work I was doing and it was arguably mine to get right in the first place.

Reply:
That would be helpful, thanks.  I've partially put it off because I was bogged down with the deep dive and slide edits, but I would be lying if the prospect of editing the message again didn't contribute slightly (but I do appreciate Vass trying to get it right).  It sounds like that was the only thing holding this up, so I will go ahead and merge so you can touch both